frontends/ol-components/src/components/Alert/Alert.tsx

```diff
@@ -7,7 +7,7 @@ import { theme } from "../ThemeProvider/ThemeProvider"
 import type { AlertProps as MuiAlertProps } from "@mui/material/Alert"
 
 type Colors = {
-  [Severity in AlertColor]: string
+  [_Severity in AlertColor]: string
```

Comment:
Just out of curiosity, what was this for?

Same question below for `_key`

Reply:
If was seeing an eslint error on the mapped type - a bug perhaps: 
```
'Severity' is defined but never used. Allowed unused vars must match /^_/u.eslint[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)
```